dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -104,11 +103,11 @@ protected String parseCommandText(String commandText) {
         return commandText;
     }
 
-    public final PreparedStatement createStatementWithBoundFixtureSymbols(
+    @Override
+    public final DbStatement createStatementWithBoundFixtureSymbols(
             TestHost testHost, String commandText) throws SQLException {
         String command = Options.isBindSymbols() ? parseCommandText(commandText) : commandText;
-        PreparedStatement cs = getConnection().prepareStatement(
-                command);
+        PreparedStatement cs = getConnection().prepareStatement(command);
```

Comment:
You did explain why `cs` was OK for a name, but I can't quite recall it now. Don't we end up creating a wrapped `PreparedStatement` from this and not a wrapped `CallableStatement`?